prototypes/process-explorer/oss-frontend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@morgan-stanley/composeui-process-explorer-frontend-oss",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build --configuration production",
+    "build-dev": "ng build --configuration development",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
```

Comment:
Currently testing hangs on CI because it's in watch mode.
Could you replace the 'test' build script by the following two:

"test": "ng test --watch=false",
"test:dev": "ng test"